### ISSUE TITLE
Running ng lint to Add Linting to an Angular 14 Project which Contains a Library Project Results in Packages Being Installed But No .eslintrc.json File or angular.json Additions

### ISSUE THREAD

**Issue by mylifeandcode** [angular-eslint/angular-eslint#1102]
I recently created a new Angular project which also contains a library project. Afterward, I ran _ng lint_ to add the linter. It successfully installed the packages and did not report any errors. However, after doing so, if I run _ng lint_ again, it just prompts me to install the packages, then tells me they're already installed. No .eslintrc.json file gets created, and no modifications are made to angular.json.

**Comment by JamesHenry** [comment]
Hi @mylifeandcode, please feel free to reopen this issue if you can provide a reproduction repo, or at least some clear steps to reproduce (in the style of the below).

I would be very happy to look into any issues which can be reproduced. You can help me to help you

---

Here is my interpretation of your description, with which I cannot reproduce any issues:

1. Create a new workspace which is focused on libraries

```
npx @angular/cli@latest new --create-application=false

? What name would you like to use for the new workspace and initial project? fourteenonethreelibrary
CREATE fourteenonethreelibrary/README.md (1077 bytes)
CREATE fourteenonethreelibrary/.editorconfig (274 bytes)
CREATE fourteenonethreelibrary/.gitignore (548 bytes)
CREATE fourteenonethreelibrary/angular.json (139 bytes)
CREATE fourteenonethreelibrary/package.json (1006 bytes)
CREATE fourteenonethreelibrary/tsconfig.json (863 bytes)
CREATE fourteenonethreelibrary/.vscode/extensions.json (130 bytes)
CREATE fourteenonethreelibrary/.vscode/launch.json (474 bytes)
CREATE fourteenonethreelibrary/.vscode/tasks.json (938 bytes)
✔ Packages installed successfully.
    Successfully initialized git.
```

2. Create a library

```
ng g library
? What name would you like to use for the library? foo
CREATE projects/foo/README.md (955 bytes)
CREATE projects/foo/karma.conf.js (1424 bytes)
CREATE projects/foo/ng-package.json (152 bytes)
CREATE projects/foo/package.json (182 bytes)
CREATE projects/foo/tsconfig.lib.json (333 bytes)
CREATE projects/foo/tsconfig.lib.prod.json (240 bytes)
CREATE projects/foo/tsconfig.spec.json (309 bytes)
CREATE projects/foo/src/public-api.ts (143 bytes)
CREATE projects/foo/src/test.ts (767 bytes)
CREATE projects/foo/src/lib/foo.module.ts (227 bytes)
CREATE projects/foo/src/lib/foo.component.spec.ts (578 bytes)
CREATE projects/foo/src/lib/foo.component.ts (256 bytes)
CREATE projects/foo/src/lib/foo.service.spec.ts (342 bytes)
CREATE projects/foo/src/lib/foo.service.ts (132 bytes)
UPDATE angular.json (1082 bytes)
UPDATE package.json (1083 bytes)
UPDATE tsconfig.json (927 bytes)
✔ Packages installed successfully.
```

3. Try and run linting and get prompted to install the angular-eslint project

```
ng lint                                                                                                                                                                                                                                                                  31s
Cannot find "lint" target for the specified project.
You can add a package that implements these capabilities.

For example:
  ESLint: ng add @angular-eslint/schematics

Would you like to add ESLint now? Yes
ℹ Using package manager: npm
✔ Found compatible package version: @angular-eslint/schematics@14.0.3.
✔ Package information loaded.

The package @angular-eslint/schematics@14.0.3 will be installed and executed.
Would you like to proceed? Yes
✔ Packages successfully installed.
    
    All @angular-eslint dependencies have been successfully installed 🎉
    
    Please see https://github.com/angular-eslint/angular-eslint for how to add ESLint configuration to your project.
    
    We detected that you have a single project in your workspace and no existing linter wired up, so we are configuring ESLint for you automatically.
    
    Please see https://github.com/angular-eslint/angular-eslint for more information.
    
CREATE .eslintrc.json (984 bytes)
CREATE projects/foo/.eslintrc.json (836 bytes)
UPDATE package.json (1460 bytes)
UPDATE angular.json (1411 bytes)
✔ Packages installed successfully.
```

4. Successfully run linting

```
ng lint                                                                                                                                                                                                                                                              х 1 17s

Linting "foo"...

/Users/james/Repos/tmp/fourteenonethreelibrary/projects/foo/src/lib/foo.component.ts
  17:3  error  Lifecycle methods should not be empty  @angular-eslint/no-empty-lifecycle-method

✖ 1 problem (1 error, 0 warnings)

Lint errors found in the listed files.
```

**Comment by dinbtechit** [comment]
I ran into the same issue. It does not create the `.eslintrc.json` file for existing [Angular Multi Project](https://angular.io/guide/file-structure#multiple-projects).

**Steps to reproduce:**
1. Create an Angular Workspace
    ```
     ng new sample-app --no-create-application
    ```
2. Generate a project and a library
    ```
    ng g application first-app --routing true 
    ng g library mylib
    ```
3.  Now add ESLint through one of the approaches:
      
     > Note: No eslintrc file was generated in both instances.

     1. Using `ng add @angular-eslint/schematics`:
          **Output:** 
         ```
         $ ng add @angular-eslint/schematics
         ℹ Using package manager: npm
         ✔ Found compatible package version: @angular-eslint/schematics@15.2.1.
         ✔ Package information loaded.

          The package @angular-eslint/schematics@15.2.1 will be installed and executed.
          Would you like to proceed? Yes
          ✔ Packages successfully installed.

          All @angular-eslint dependencies have been successfully installed 🎉

          Please see https://github.com/angular-eslint/angular-eslint for how to add ESLint configuration to your project.

          UPDATE package.json (1447 bytes)
         ✔ Packages installed successfully.
         ```
     2. Using `ng lint`
         **Output**
         ```
         $ ng lint
         Cannot find "lint" target for the specified project.
         You can add a package that implements these capabilities.

         For example:
         ESLint: ng add @angular-eslint/schematics

         Would you like to add ESLint now? Yes
         ℹ Using package manager: npm
         ✔ Found compatible package version: @angular-eslint/schematics@15.2.1.
         ✔ Package information loaded.

         The package @angular-eslint/schematics@15.2.1 will be installed and executed.
         Would you like to proceed? Yes
        ✔ Packages successfully installed.

         All @angular-eslint dependencies have been successfully installed 🎉

         Please see https://github.com/angular-eslint/angular-eslint for how to add ESLint configuration to your project.

         UPDATE package.json (1447 bytes)
        ✔ Packages installed successfully.
     
         ```
